plugins/vli/fu-vli-pd-common.c

```diff
@@ -20,15 +20,19 @@ fu_vli_pd_common_guess_device_kind(guint32 fwver)
 		return FU_VLI_DEVICE_KIND_VL101;
 	if (tmp == 0x07 || tmp == 0x08)
 		return FU_VLI_DEVICE_KIND_VL102;
-	if (tmp == 0x09)
+	if (tmp == 0x09 || tmp == 0x0a)
 		return FU_VLI_DEVICE_KIND_VL103;
 	if (tmp == 0x0b)
 		return FU_VLI_DEVICE_KIND_VL104;
-	if (tmp == 0xa0)
+	if (tmp == 0x0c)
 		return FU_VLI_DEVICE_KIND_VL105;
-	if (tmp == 0xb0)
+	if (tmp == 0x0d)
```

Comment:
can we get some #defines for these? Given this has caused at least one regression it would be good to test each chip type in the self tests too.

Reply:
How about 
#define FU_VLI_DEVICE_FW_TAG_VL103A     0x09
#define FU_VLI_DEVICE_FW_TAG_VL103B     0x0A
#define FU_VLI_DEVICE_FW_TAG_VL104       0x0B
#define FU_VLI_DEVICE_FW_TAG_VL105       0x0C
...

Reply:
Sounds good to me

Reply:
Also, does an enum make more sense than defines?

Reply:
maybe, because VL108 (0xa1/0xb1) and  VL109 (0xa2/0xb2) need to assign value.
And, the compare byte need to change also, "if (tmp<0xa0) tmp = tmp&0x0F;"
Too many chip versions~